crates/yttrium/src/bundler/client.rs

```diff
@@ -298,6 +298,7 @@ mod tests {
                 max_priority_fee_per_gas: max_priority_fee_per_gas.into(),
                 paymaster: paymaster,
                 paymaster_data: paymaster_data,
+                // authorization_list: None,
```

Comment:
I'm assuming this is going to be used in future PRs?

Reply:
Yeah possibly